tests/widgets_chart.rs

```diff
@@ -203,7 +203,7 @@ fn widgets_chart_handles_long_labels<'line, Lines>(
         "          ",
         "          ",
         "──────────",
-        "AAA  B   C",
+        "AAA B    C",
```

Comment:
This is better aligned with how line treats centering (ambiguous centering preferences to the left)